cognee/modules/retrieval/cypher_query_retriever.py

```
@@ -0,0 +1,77 @@
+        )
+        previous_attempt_context = None
+        for attempt in range(self.max_attempts):
+            llm_client = get_llm_client()
+            system_prompt = f"""
+    You are an advanced Cypher query generator for Neo4j. Your task:
+    - Based on the following graph schema (nodes and edges) and a natural language query, generate a valid Cypher query that retrieves the requested information.
+    - Provide only the Cypher query, no explanations or additional formatting.
+    - Take into account the previous failed attempts and adjust the query accordingly.
+    
+    Return only a valid, optimized Cypher query.
+    """
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)